verifiers/utils/git_test.go

```diff
@@ -0,0 +1,244 @@
+package utils
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	serrors "github.com/slsa-framework/slsa-verifier/v2/errors"
+)
+
+func Test_NormalizeGitURI(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		name     string
+		uri      string
+		expected string
+	}{
+		{
+			name:     "empty uri",
```

Comment:
do we need a test with a `@ref` as well? Or it's only relevant for `ParseGitURIAndRef`?

Reply:
It's not really relevant in this case since the it doesn't look at the ref, but I added the tests just in case.